frontend/views/containers/chatroom/CreateNewChannelModal.vue

```diff
@@ -62,19 +66,23 @@
         i18n.is-outlined(tag='button' @click='close') Cancel
         i18n.is-success(
           tag='button'
+          data-test='createChannelSubmit'
```

Comment:
Pressing "Enter" key when entering the channel name dismisses the modal instead of creating the channel.

<img width="547" alt="Screen Shot 2022-04-21 at 2 11 37 PM" src="https://user-images.githubusercontent.com/138706/164553125-654c76e2-0ff2-49fe-8af2-bf608b86052b.png">

Reply:
@taoeffect , I think this issue is related to #1155.
We need to check all the forms not only this form inside `CreateChannelModal`.

Reply:
OK, we can leave it for later then